src/pages/contact.astro

```diff
@@ -0,0 +1,299 @@
+---
+import Layout from "../layouts/Layout.astro";
+const pageTitle = "Contact Us - UBC iGEM";
+---
+
+<Layout title={pageTitle}>
+  <style>
+    @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700;900&family=Open+Sans:wght@400;700&display=swap");
+
+    h1 {
+      font-family: Poppins;
+      font-size: 4.5em;
+      font-weight: 900;
+      text-decoration-color: #042a2b;
+    }
+    h2 {
+      font-size: 2em;
+      font-family: "Open Sans";
+      font-weight: 400;
+      margin-top: 0.5em;
+      margin-bottom: 0.5em;
+      text-decoration-color: #042a2b;
+    }
+    p {
+      font-size: 1.3em;
+      font-family: "Open Sans";
+      text-decoration-color: #042a2b;
+    }
+
+    .contact-section {
+      background-color: #afdbbf;
+      padding-bottom: 5%;
+      padding-inline: 10%;
+      text-align: left;
+      border-top: solid 2.5px black;
+      border-bottom: solid 2.5px black;
+      text-decoration-color: #042a2a;
+      outline: black;
+      max-width: 100%;
+    }
+
+    .contact-row {
+      display: flex;
+      align-items: center;
+    }
+
+    .faq-section {
+      padding-inline: 10%;
+      padding-bottom: 5%;
+      padding-top: 5%;
+      border-bottom: solid 2.5px black;
+      text-align: left;
+      max-width: 100%;
+    }
+
+    .acknowledgement-section {
+      display: flex-col;
+      padding: 40px 10%;
+      text-align: left;
+      font-family: "Open Sans", sans-serif;
+      max-width: 100%;
+    }
+
+    .acknowledgement-text {
+      flex: 2;
+      font-size: 1.1rem;
+      line-height: 1.6;
+      color: #333;
+      margin-bottom: 50px;
+    }
+
+    .footer-content {
+      flex: 3; /* Adjust flex proportion as needed */
+      display: flex;
+      justify-content: space-between; /* Push social icons to the right */
+      align-items: flex-start;
+    }
+
+    .footer-main-area {
+      /* New class for grouping logo and nav */
+      display: flex;
+      align-items: flex-start;
+      gap: 30px; /* Space between logo and nav */
+    }
+
+    .footer-nav ul {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      margin: 0;
+      flex-direction: column;
+      gap: 10px;
+    }
+
+    .footer-nav a {
+      font-family: "Open Sans", sans-serif;
+      font-weight: 500;
+      font-size: 16px;
+      color: #333333;
+      text-decoration: none;
+      transition: color 0.2s ease;
+    }
+
+    .footer-nav a:hover {
+      text-decoration: underline;
+    }
+
+    .social-icons {
+      display: flex;
+      justify-content: flex-end;
+      gap: 16px;
+      align-items: flex-start;
+    }
+
+    .social-icons img {
+      width: 48px;
+      height: 48px
+    }
+  </style>
+
+  <section class="contact-section">
+    <h1 style="margin-bottom: 0.75em">Contact us</h1>
+
+    <div class="contact-row">
+      <svg width="72" height="72" fill="none" xmlns="http://www.w3.org/2000/svg" style="flex-shrink:0;">
```

Comment:
why are you inlining the svgs here if we're already adding them to the repo and loading them onto the page? just use the svgs in `public/`.